app/routers/impulso_previne.py

```diff
@@ -1,162 +1,173 @@
-    return res
-
-@router.post("/impulsoprevine/capacitacao/avaliacao-conteudo")
-async def avaliacao(usuario_id: str = Form(...), codigo_conteudo: str = Form(...),avaliacao: int = Form(...),username: Usuario = Depends(auth.get_current_user)):
-    res = TrilhaCapacitacao.avaliacao_conteudo(usuario_id,codigo_conteudo,avaliacao)
-    return res
-
-@router.post("/impulsoprevine/capacitacao/consulta-avaliacao-conclusao")
-async def consulta_avaliacao_conclusao_conteudo(
-    usuario_id: str = Form(...),
-    codigo_conteudo: str = Form(...),
-    username: Usuario = Depends(auth.get_current_user)
-    ):
-    res = TrilhaCapacitacao.consulta_avaliacao_conclusao(usuario_id,codigo_conteudo)
-    return res
-
-@router.post("/impulsoprevine/capacitacao/consulta-avaliacao-conclusao-por-usuario")
-async def consulta_avaliacao_conclusao_conteudo(
-    usuario_id: str = Form(...),
-    username: Usuario = Depends(auth.get_current_user)
-    ):
-    res = TrilhaCapacitacao.consulta_avaliacao_conclusao_por_usuario(usuario_id)
-    return res
+from fastapi.security import OAuth2PasswordRequestForm
+from app.controllers import indicadores,forum_ip,auth,busca_ativa_gestantes,busca_ativa_diabeticos,busca_ativa_hipertensos,TrilhaCapacitacao, indicadores_desempenho_score_equipes_validas,indicadores_municipios_equipes_homologadas
+from fastapi import APIRouter, Depends,Form
+from typing import Optional, List
+from pydantic import BaseModel
+from app.controllers.auth import get_current_user,Usuario
+
+router = APIRouter()
+
+class Indicador(BaseModel):
+    id : int
+    estado_sigla : str
+    estado_nome : str
+    estado_nome_normalizado : str
+    municipio_id_sus : str
+    municipio_nome : str
+    municipio_nome_normalizado : str
+    indicadores_parametros_id : int
+    indicadores_parametros_nome : str
+    indicadores_parametros_nome_normalizado : str
+    indicadores_parametros_ordem : int
+    indicadores_parametros_meta : float
+    indicadores_parametros_peso : float
+    indicadores_resultados_porcentagem : float
+    diff_numerador_para_meta : int
+
+    class Config:
+        orm_mode = True
+
+@router.get("/impulsoprevine/indicadores/resumo", response_model=List[Indicador])
+async def consulta_indicadores(indicadores_parametros_id: Optional[str] = None, indicadores_nome: Optional[str] = None , estado_sigla: Optional[str] = None, estado_nome: Optional[str] = None, id_sus: Optional[str] = None, municipio_nome: Optional[str] = None):
+    res = indicadores.consulta_indicadores(id_sus,municipio_nome,estado_sigla,estado_nome,indicadores_nome,indicadores_parametros_id)
+    return res
+#aquii
+@router.get("/impulsoprevine/indicadores/municipios_equipes_homologadas")
+async def consulta_indicadores_equipes_homologadas_municipios(municipio_uf: Optional[str] = None):
+    res = indicadores_municipios_equipes_homologadas.consulta_indicadores_municipios_equipes_homologadas(municipio_uf)

```

Comment:
O que acha de renomear essas funções com verbos?
```suggestion
async def consultar_indicadores_equipes_homologadas_municipios(municipio_uf: Optional[str] = None):
    res = indicadores_municipios_equipes_homologadas.consultar_indicadores_municipios_equipes_homologadas(municipio_uf)

```